py/code.go

```diff
@@ -167,6 +164,38 @@ func NewCode(argcount int32, kwonlyargcount int32,
 			}
 		}
 	}
+
+	return MakeCode(argcount,
+		kwonlyargcount,
+		nlocals,
+		stacksize,
+		flags,
+		code,
+		consts,
+		names,
+		varnames,
+		freevars,
+		cellvars,
+		filename,
+		name,
+		firstlineno,
+		lnotab)
+}
+
+// Make a new code object
```

Comment:
having both `MakeCode` and `NewCode` with the same doc and signature, this needs a meaningful documentation string, explaining what's the difference between the two.